screens/InventoryScreen.js

```diff
@@ -40,7 +40,7 @@ class InventoryScreen extends React.Component {
                         <View style={[styles.menuContainer]}>
                             <MenuButton
                                 route='InventoryOrderScreen'
-                                onPress={() => NavigationService?.navigateToRoute('InventoryOrderScreen')}
+                                onPress={() => NavigationService?.navigateToRoute('InventoryOrderScreen', {sreen: 'InventoryOrderScreen'})}
```

Comment:
Is there a typo? sreen??

And why are we passing a {screen: '<screen name>'} in the second parameter?

Reply:
Yes, it's a typo. I fix it as 'screen'.